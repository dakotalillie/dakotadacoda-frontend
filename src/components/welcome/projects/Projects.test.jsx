import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Projects from './Projects';

describe('Projects', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Projects />);
  });

  it('renders successfully', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});