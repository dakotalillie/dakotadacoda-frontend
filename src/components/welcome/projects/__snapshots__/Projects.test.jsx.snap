// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders successfully 1`] = `
<div
  className="projects section grey-bg"
>
  <Container
    tag="div"
  >
    <h2>
      Recent Projects
    </h2>
    <CardDeck
      tag="div"
    >
      <Card
        key="1"
        tag="div"
      >
        <div
          className="image_container"
        >
          <CardImg
            alt="Stratagem Cover"
            src="diplomacy.png"
            tag="img"
            top={true}
            width="100%"
          />
          <div
            className="image_overlay"
          />
        </div>
        <CardBody
          tag="div"
        >
          <CardTitle
            tag="h5"
          >
            Stratagem
          </CardTitle>
          <CardSubtitle
            tag="h6"
          >
            Online Board Game
          </CardSubtitle>
          <CardText
            tag="p"
          >
            Built Using:
          </CardText>
          <Row
            tag="div"
          >
            <Col
              className="tech_row"
              key="React"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="React"
                src="react.png"
                url="https://reactjs.org/"
              />
            </Col>
            <Col
              className="tech_row"
              key="Redux"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="Redux"
                src="redux_transparent.png"
                url="https://redux.js.org/"
              />
            </Col>
            <Col
              className="tech_row"
              key="Bootstrap"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="Bootstrap"
                src="bootstrap.png"
                url="https://getbootstrap.com/"
              />
            </Col>
            <Col
              className="tech_row"
              key="Django"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="Django"
                src="django.png"
                url="https://www.djangoproject.com/"
              />
            </Col>
            <Col
              className="tech_row"
              key="PostgreSQL"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="PostgreSQL"
                src="postgresql.png"
                url="https://www.postgresql.org/"
              />
            </Col>
            <Col
              className="tech_row"
              key="Jest"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="Jest"
                src="jest.png"
                url="http://jestjs.io/"
              />
            </Col>
          </Row>
        </CardBody>
        <CardFooter
          tag="div"
        >
          <Row
            tag="div"
          >
            <Col
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <Button
                color="link"
                href="https://github.com/dakotalillie/stratagem-frontend"
                tag="button"
              >
                <img
                  alt="Github"
                  className="github_icon"
                  src="github.png"
                />
                 
                <span>
                  front
                </span>
              </Button>
            </Col>
            <Col
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <Button
                color="link"
                href="https://github.com/dakotalillie/stratagem_api"
                tag="button"
              >
                <img
                  alt="Github"
                  className="github_icon"
                  src="github.png"
                />
                 
                <span>
                  back
                </span>
              </Button>
            </Col>
          </Row>
        </CardFooter>
      </Card>
      <Card
        key="0"
        tag="div"
      >
        <div
          className="image_container"
        >
          <CardImg
            alt="BeetBox Cover"
            src="beetbox.png"
            tag="img"
            top={true}
            width="100%"
          />
          <div
            className="image_overlay"
          />
        </div>
        <CardBody
          tag="div"
        >
          <CardTitle
            tag="h5"
          >
            BeetBox
          </CardTitle>
          <CardSubtitle
            tag="h6"
          >
            Sample Library Manager
          </CardSubtitle>
          <CardText
            tag="p"
          >
            Built Using:
          </CardText>
          <Row
            tag="div"
          >
            <Col
              className="tech_row"
              key="React"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="React"
                src="react.png"
                url="https://reactjs.org/"
              />
            </Col>
            <Col
              className="tech_row"
              key="Redux"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="Redux"
                src="redux_transparent.png"
                url="https://redux.js.org/"
              />
            </Col>
            <Col
              className="tech_row"
              key="Bootstrap"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="Bootstrap"
                src="bootstrap.png"
                url="https://getbootstrap.com/"
              />
            </Col>
            <Col
              className="tech_row"
              key="Ruby On Rails"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="Ruby On Rails"
                src="rails.png"
                url="http://rubyonrails.org/"
              />
            </Col>
            <Col
              className="tech_row"
              key="PostgreSQL"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="PostgreSQL"
                src="postgresql.png"
                url="https://www.postgresql.org/"
              />
            </Col>
            <Col
              className="tech_row"
              key="AWS S3"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="AWS S3"
                src="s3.png"
                url="https://aws.amazon.com/s3/"
              />
            </Col>
          </Row>
        </CardBody>
        <CardFooter
          tag="div"
        >
          <Row
            tag="div"
          >
            <Col
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <Button
                color="link"
                href="https://github.com/dakotalillie/beetbox-dom"
                tag="button"
              >
                <img
                  alt="Github"
                  className="github_icon"
                  src="github.png"
                />
                 
                <span>
                  front
                </span>
              </Button>
            </Col>
            <Col
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <Button
                color="link"
                href="https://github.com/dakotalillie/beetbox-api"
                tag="button"
              >
                <img
                  alt="Github"
                  className="github_icon"
                  src="github.png"
                />
                 
                <span>
                  back
                </span>
              </Button>
            </Col>
          </Row>
        </CardFooter>
      </Card>
      <Card
        key="2"
        tag="div"
      >
        <div
          className="image_container"
        >
          <CardImg
            alt="FlatChat Cover"
            src="flatchat.png"
            tag="img"
            top={true}
            width="100%"
          />
          <div
            className="image_overlay"
          />
        </div>
        <CardBody
          tag="div"
        >
          <CardTitle
            tag="h5"
          >
            FlatChat
          </CardTitle>
          <CardSubtitle
            tag="h6"
          >
            Realtime Messaging Service
          </CardSubtitle>
          <CardText
            tag="p"
          >
            Built Using:
          </CardText>
          <Row
            tag="div"
          >
            <Col
              className="tech_row"
              key="React"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="React"
                src="react.png"
                url="https://reactjs.org/"
              />
            </Col>
            <Col
              className="tech_row"
              key="Redux"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="Redux"
                src="redux_transparent.png"
                url="https://redux.js.org/"
              />
            </Col>
            <Col
              className="tech_row"
              key="Bootstrap"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="Bootstrap"
                src="bootstrap.png"
                url="https://getbootstrap.com/"
              />
            </Col>
            <Col
              className="tech_row"
              key="Ruby On Rails"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="Ruby On Rails"
                src="rails.png"
                url="http://rubyonrails.org/"
              />
            </Col>
            <Col
              className="tech_row"
              key="PostgreSQL"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <TechIcon
                alt="PostgreSQL"
                src="postgresql.png"
                url="https://www.postgresql.org/"
              />
            </Col>
          </Row>
        </CardBody>
        <CardFooter
          tag="div"
        >
          <Row
            tag="div"
          >
            <Col
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <Button
                color="link"
                href="https://github.com/dakotalillie/flatchat_dom"
                tag="button"
              >
                <img
                  alt="Github"
                  className="github_icon"
                  src="github.png"
                />
                 
                <span>
                  front
                </span>
              </Button>
            </Col>
            <Col
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
              xs="4"
            >
              <Button
                color="link"
                href="https://github.com/dakotalillie/flatchat_api"
                tag="button"
              >
                <img
                  alt="Github"
                  className="github_icon"
                  src="github.png"
                />
                 
                <span>
                  back
                </span>
              </Button>
            </Col>
          </Row>
        </CardFooter>
      </Card>
    </CardDeck>
  </Container>
</div>
`;
